bundles/org.openhab.binding.surepetcare/src/main/resources/ESH-INF/thing/things.xml

```diff
@@ -0,0 +1,619 @@
+		<item-type>String</item-type>
+		<label>LED Mode</label>
+		<description>The current LED mode of the device</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">0</option>
+				<option value="1">1</option>
+				<option value="4">4</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="pairingModeType" advanced="true">
+		<item-type>String</item-type>
+		<label>Pairing Mode</label>
+		<description>The current pairing mode of the device</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">0</option>
+				<option value="1">1</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="onlineType">
+		<item-type>Switch</item-type>
+		<label>Online</label>
+		<description>Indicator if device is online or offline</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+
+	<channel-type id="curfewEnabledType">
+		<item-type>Switch</item-type>
+		<label>Curfew Enabled</label>
+		<description>Indicator if the curfew is enabled</description>
+		<state readOnly="false" pattern="%s" />
+	</channel-type>
+
+	<channel-type id="curfewLockTimeType">
+		<item-type>String</item-type>
+		<label>Curfew Lock Time</label>
+		<description>The time when the curfew starts</description>
+		<state readOnly="false" pattern="%s" />
+	</channel-type>
+
+	<channel-type id="curfewUnlockTimeType">
+		<item-type>String</item-type>
+		<label>Curfew Unlock Time</label>
+		<description>The time when the curfew finishes</description>
+		<state readOnly="false" pattern="%s" />
+	</channel-type>
+
+	<channel-type id="dateOfBirthType">
+		<item-type>DateTime</item-type>
+		<label>Pet Date of Birth</label>
+		<description>The pet's date of birth.</description>
+		<category>Time</category>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td" />
+	</channel-type>
+
+	<channel-type id="weightType">
+		<item-type>Number:Mass</item-type>
+		<label>Pet Weight</label>
+		<description>The pet weight</description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+
+	<channel-type id="feederDeviceType">
+		<item-type>String</item-type>
+		<label>Pet Feeding Device Name</label>
+		<description>The pet feeding device name</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+
+	<channel-type id="feederLastFeedingType">
+		<item-type>DateTime</item-type>
+		<label>Pet Last Feeding</label>
+		<description>The last pet feeding</description>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS" />
+	</channel-type>
+
+	<channel-type id="feederLastChangeType">
+		<item-type>Number:Mass</item-type>
+		<label>Pet Feeding Last Change</label>
+		<description>The pet feeding last change</description>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+
+	<channel-type id="feederLastChangeLeftType">
+		<item-type>Number:Mass</item-type>
+		<label>Pet Feeding Last Change Left</label>
```

Comment:
I fear i dont understand you comment...

This value is for the pet feeder device... it has a scale included... so if the pet eat and left the device... the change in gram is reported.

Reply:
> data 

Uhm, sorry I meant 'date'. I thought the label sounded like a date but the unit suggested otherwise. If you have the idea that you could improve the title that would of course be great.